src/doc-utils.js

```diff
@@ -33,6 +33,23 @@ function traverseDoc(doc, onEnter, onExit) {
   traverseDocRec(doc);
 }
 
+function mapDoc(doc, func) {
+  doc = func(doc);
+
+  if(doc.type === "concat") {
```

Comment:
It's a bit different; `removeLines` doesn't have any `else if`s. I personally prefer being explicit that it's all in the same branch logic.

I've always preferred this style:

```
if(code) {
  lotsOfComplicatedCode()
}
else if(otherComplications()) {
  evenMoreComplications()
}
```

because I think it makes it *much* clearer to read but for some reason the JS community has standardized more or less on the above code which is why prettier formats it that way.